api/client/build.go

```diff
@@ -131,11 +130,19 @@ func (cli *DockerCli) CmdBuild(args ...string) error {
 		return fmt.Errorf("cannot canonicalize dockerfile path %s: %v", relDockerfile, err)
 	}
 
-	var includes = []string{"."}
+	var (
```

Comment:
I'm okay with `var` words, just includes should be defined below where it used first time.